typeclasses/src/main/scala/Ordering.scala

```diff
@@ -38,6 +38,18 @@ object Ordering extends App:
     case Order.LT => a
     case Order.EQ => a
 
+  def bubbleSort[A: Ordering](as: List[A]): List[A] = {
+    def swap(myAs: List[A]): List[A] =
+      myAs match
+        case a :: b :: rest if a.compare(b) == Order.GT => b :: swap(a :: rest)
```

Comment:
I would avoid using guards when not strictly necessary, as it disables exhaustivity checking.